sdk/include/reactos/wine/ddk/imm.h

```diff
@@ -96,13 +96,15 @@ typedef struct INPUTCONTEXTDX /* unconfirmed */
     DWORD dwUIFlags;
     DWORD dwUnknownDog;
     void *pUnknownFox;
-    /* ... */
+    DWORD dwUnknownPig;
+    DWORD dwUnknownPony;
```

Comment:
come on.... Please use standard names `dwUnknown1`, `dwUnknown2` and so on.

Reply:
OK.